src/swipe.js

```diff
@@ -1,537 +0,0 @@
-/**
```

Comment:
Yeah. It was deprecated for a long time. No longer used. Easier to delete now than to refactor.